backend-project/small_eod/generic/tests.py

```diff
@@ -74,6 +74,28 @@ def get_create_data(self):
         return data
 
 
+class ReadOnlyPaginatedViewSetMixin(ReadOnlyViewSetMixin):
```

Comment:
Skoro domyślnie dla wszystkich list w projekcie zmieniłeś, aby wyniki były paginowane (powstaje reguła) to dlaczego nie zmodyfikować ```GenericViewSetMixin```, aby obsługiwał paginacje? W CBV Django włączenie paginacji wymaga wyłącznie ustalenia właściwości ```paginated_by```, tak w tym przypadku myślę, że wystarczy pole ```paginated=True``` , a następnie odpowiednie dostosowanie ```test_list_plain```.